### PR TITLE
Talbpaul/rework maxqsize

### PR DESCRIPTION
**Pull request by PaulTalbot-INL**
--------
Pull Request Description
--------
Fixes max queue size to be consistent with what we claim in the manual (default to batch size).

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added?

### CODE REVIEW

File: tests/framework/Samplers/DynamicEventTrees/testDETshort.xml [hunk 1]
```diff
@@ -18,12 +18,14 @@
       <revision author="alfoa" date="2016-10-20">Modified XSD and removed dim attribute from all the tests</revision>
       <revision author="alfoa" date="2017-01-21">Adding this test description.</revision>
       <revision author="alfoa" date="2018-01-24">Modification for new DataObject and new option to specify printing and plotting subdirectories (attribute ``dir'')</revision>
+      <revision author="alfoa" date="2018-01-26">Added maxqueuesize 1 to be consistent with historical behavior</revision>
```
Comment (alfoa) [inline tests/framework/Samplers/DynamicEventTrees/testDETshort.xml]: I do not think I did this modification :D

Comment (PaulTalbot-INL) [reply tests/framework/Samplers/DynamicEventTrees/testDETshort.xml]: My bad.

**Comment by PaulTalbot-INL** [comment]
This has passed the available tests and approved by 2 reviewers.  Can it be merged?

**Comment by alfoa** [comment]
There is no issue associated with this PR... can you open one for tracking purposes?

**Comment by alfoa** [comment]
Ops.
Sorry....never mind... it goes to the dataobject-rework. No issue required

**Comment by alfoa** [comment]
----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [ ] 1. Review all computer code. ***ok***
- [ ] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts). ***n/a***
- [ ] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details. ***ok***
- [ ] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass. ***ok***
- [ ] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True. ***n/a***
- [ ] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync. ***n/a***
- [ ] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done. ***no issue created since this is part of the general DataObject restructuring effort***
- [ ] 8. If an analytic test is changed/added is the the analytic documentation updated/added? ***n/a***

**Pull request merged**